### ISSUE TITLE
Crash on No static method getInstance()Lcom/braintreepayments/api/BraintreeSharedPreferences

### ISSUE THREAD

**Issue by mtrakal** [braintree/braintree-android-drop-in#371]
### General information

* SDK/Library version: Braintree android: 4.20.0, dropin: 6.4.0
* Environment: Both, android app crash
* Android Version and Device: Pixel 3a (Android 12)

### Issue description

```
    java.lang.NoSuchMethodError: No static method getInstance()Lcom/braintreepayments/api/BraintreeSharedPreferences; in class Lcom/braintreepayments/api/BraintreeSharedPreferences; or its super classes (declaration of 'com.braintreepayments.api.BraintreeSharedPreferences' appears in /data/app/~~haUIFXrUblW8IbIVpgU40w==/cz.nom.smilecarweb.debug-4LTD7EgfrRFVcZfTZ_0drw==/base.apk)
        at com.braintreepayments.api.DropInSharedPreferences.<init>(DropInSharedPreferences.java:30)
        at com.braintreepayments.api.DropInSharedPreferences.getInstance(DropInSharedPreferences.java:19)
        at com.braintreepayments.api.DropInClient.createDefaultParams(DropInClient.java:56)
        at com.braintreepayments.api.DropInClient.<init>(DropInClient.java:161)
```

App is upgraded from libs versions:
```
com-braintreepayments = "4.15.0"

com-braintreepayments-api-card = { module = "com.braintreepayments.api:card", version.ref = "com-braintreepayments" }
com-braintreepayments-api-data-collector = { module = "com.braintreepayments.api:data-collector", version.ref = "com-braintreepayments" }
com-braintreepayments-api-drop-in = "com.braintreepayments.api:drop-in:6.4.0"
com-braintreepayments-api-google-pay = { module = "com.braintreepayments.api:google-pay", version.ref = "com-braintreepayments" }
com-braintreepayments-api-local-payment = { module = "com.braintreepayments.api:local-payment", version.ref = "com-braintreepayments" }
com-braintreepayments-api-paypal = { module = "com.braintreepayments.api:paypal", version.ref = "com-braintreepayments" }
com-braintreepayments-api-three-d-secure = { module = "com.braintreepayments.api:three-d-secure", version.ref = "com-braintreepayments" }

com.braintreepayments.api:drop-in:6.3.0"
```

If I use com-braintreepayments = "4.19.0" looks, that works ok.

**Comment by mtrakal** [comment]
looks relevant to: https://github.com/braintree/braintree_android/issues/619

**Comment by sshropshire** [comment]
Hi @mtrakal thanks for using the Braintree SDK for Android. We still need to update the DropIn library to the latest 4.20.0 version. In this case, it appears that Gradle is choosing the newest version over the one bundled with 4.19.0.

In the meantime as a workaround, you can use `4.19.0` until we release an update to DropIn.

**Comment by sshropshire** [comment]
Hey @mtrakal

**Comment by sshropshire** [comment]
@mtrakal this is now available in version [6.5.0](https://github.com/braintree/braintree-android-drop-in/blob/master/CHANGELOG.md#650).
